examples/mcmc_examples/PyFstat_example_semi_coherent_MCMC_search.py

```diff
@@ -90,3 +90,8 @@
 mcmc.plot_corner(add_prior=True, truths=signal_parameters)
 mcmc.plot_prior_posterior(injection_parameters=signal_parameters)
 mcmc.plot_chainconsumer(truth=signal_parameters)
+mcmc.plot_cumulative_max(
+    plot_label="MCMC Example",
+    # title="Cumulative 2F for the best MCMC candidate",
+    custom_axis_kwargs={"title": "Cumulative 2F for the best MCMC candidate"},
```

Comment:
As demonstrated by this previously broken example, from the user perspective it makes no sense to have a direct kwarg for the `plot_label` but have to put the overall title into an esoteric `custom_axis_kwargs`...

Reply:
That's right: I just followed the inertia of the code. I'll include it inside `custom_axis_kwargs` and rename it to `custom_axes_kwargs`.

Reply:
Actually, `custom_ax_kwargs` makes more sense. Done in db86df05aea35c6a769c708683f0b2ba4a5cb565.